mixnet/client/src/config.rs

```diff
@@ -11,7 +11,9 @@ pub struct MixnetClientConfig {
     pub mode: MixnetClientMode,
     pub topology: MixnetTopology,
     pub connection_pool_size: usize,
+    #[serde(default = "MixnetClientConfig::default_max_retries")]
     pub max_retries: usize,
+    #[serde(default = "MixnetClientConfig::default_retry_delay")]
```

Comment:
Is this from another branch?

Reply:
No, the another branch had defaults for `MixnetNode`, I've added these for `MixnetClient` config parsing.